### PR TITLE
LP-2744 - Send adg password change success email

### PR DESCRIPTION
**Pull request by MunamMubashir-arbisoft**
[Ticket: LP-2744](https://philanthropyu.atlassian.net/browse/LP-2744)

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2144/workflows/workflows_id/jobs/2144/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (100%)
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/constants.py (100%)
openedx/adg/lms/applications/helpers.py (100%)
openedx/adg/lms/applications/models.py (86.7%): Missing lines 106,109
openedx/adg/lms/applications/rules.py (100%)
openedx/adg/lms/applications/tests/constants.py (100%)
openedx/adg/lms/applications/tests/factories.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py (100%)
openedx/adg/lms/applications/tests/test_helpers.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
openedx/adg/lms/courseware_override/tests/test_helpers.py (100%)
openedx/adg/lms/student/helpers.py (100%)
openedx/adg/lms/student/tests/test_helpers.py (100%)
-------------
Total:   169 lines
Missing: 2 lines
Coverage: 98%
-------------
```

File: openedx/core/djangoapps/user_authn/views/password_reset.py [hunk 1]
```diff
@@ -516,7 +519,11 @@ def post(self, request, *args, **kwargs):
         if password_reset_successful and is_account_recovery:
             self._handle_password_creation(request, updated_user)
 
-        send_password_reset_success_email(updated_user, request)
+        if is_testing_environment():
+            send_password_reset_success_email(updated_user, request)
+        else:
+            compose_and_send_adg_password_reset_success_email(updated_user)
```
Comment (asadali145) [inline openedx/core/djangoapps/user_authn/views/password_reset.py:525]: Lets please follow the way used in this method `adg.lms.student.helpers.send_account_activation_email`.
Create new method in adg directory and use it in place of `send_password_reset_success_email ` and that method will call `send_password_reset_success_email ` if is_testing_env else use our adg logic. This way we will have minimum core changes.

File: openedx/core/djangoapps/user_authn/views/password_reset.py [hunk 2]
```diff
@@ -741,7 +748,12 @@ def password_reset_logistration(request, **kwargs):
                         }
                     )
                     user.save()
-                    send_password_reset_success_email(user, request)
+
+                    if is_testing_environment():
+                        send_password_reset_success_email(user, request)
+                    else:
+                        compose_and_send_adg_password_reset_success_email(user)
+
```
Comment (asadali145) [inline openedx/core/djangoapps/user_authn/views/password_reset.py:756]: Same here. Create and use the method mentioned above.

**Review by asadali145 — CHANGES_REQUESTED** [review]
Added a suggestion, Have a look pls...

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/2148/workflows/workflows_id/jobs/2148/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (100%)
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/constants.py (100%)
openedx/adg/lms/applications/helpers.py (100%)
openedx/adg/lms/applications/models.py (86.7%): Missing lines 106,109
openedx/adg/lms/applications/rules.py (100%)
openedx/adg/lms/applications/tests/constants.py (100%)
openedx/adg/lms/applications/tests/factories.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py (100%)
openedx/adg/lms/applications/tests/test_helpers.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
openedx/adg/lms/courseware_override/tests/test_helpers.py (100%)
openedx/adg/lms/student/helpers.py (75.0%): Missing lines 80-81
openedx/adg/lms/student/tests/test_helpers.py (100%)
-------------
Total:   175 lines
Missing: 4 lines
Coverage: 97%
-------------
```

File: openedx/core/djangoapps/user_authn/views/password_reset.py [hunk 3]
```diff
@@ -28,7 +28,10 @@
 from rest_framework.views import APIView
 
 from common.djangoapps.edxmako.shortcuts import render_to_string
-from openedx.adg.lms.student.helpers import compose_and_send_adg_password_reset_email
+from openedx.adg.lms.student.helpers import (
+    compose_and_send_adg_password_reset_email,
+    send_adg_password_reset_success_email
```
Comment (asadali145) [inline openedx/core/djangoapps/user_authn/views/password_reset.py:33]: Pls update imports. There is an un-used import.

**Review by asadali145 — CHANGES_REQUESTED** [review]

File: openedx/core/djangoapps/user_authn/views/password_reset.py (hunk 3 above)
Comment (MunamMubashir-arbisoft) [reply openedx/core/djangoapps/user_authn/views/password_reset.py:33]: No bro that is being used to send reset email. I added the import for reset success email

Comment (asadali145) [reply openedx/core/djangoapps/user_authn/views/password_reset.py:33]: Oooopsss... Sorry. My bad!

**Review by asadali145 — DISMISSED** [review]
LGTM, Nice Work

**Review by javeriaejaz-arbisoft — APPROVED** [review]

**Review by asadali145 — APPROVED** [review]

**Review by javeriaejaz-arbisoft — APPROVED** [review]

**Comment by MunamMubashir-arbisoft** [comment]
The quality action error is due the core file changes.

**Pull request merged**